smithy-aws-endpoints/src/test/java/software/amazon/smithy/rulesengine/aws/traits/StandardPartitionalEndpointsTraitTest.java

```diff
@@ -27,6 +27,10 @@ public void loadsFromModel() {
 
         assertEquals(trait.getEndpointPatternType(), EndpointPatternType.SERVICE_DNSSUFFIX);
         assertEquals(trait.getPartitionEndpointSpecialCases().size(), 0);
+        assertEquals(trait,
+                new StandardPartitionalEndpointsTrait.Provider()
+                        .createTrait(StandardPartitionalEndpointsTrait.ID,
+                                trait.toBuilder().sourceLocation(trait.getSourceLocation()).build().toNode()));
```

Comment:
Instead of all these `.sourceLocation` sets for both traits, the traits should probably set it in the `toBuilder` call themselves.

Reply:
done!